lib/explorer/data_frame.ex

```diff
@@ -631,32 +605,33 @@ defmodule Explorer.DataFrame do
   @spec arrange(
           df :: DataFrame.t(),
           columns ::
-            [String.t() | {String.t(), :asc | :desc}]
-            | Keyword.t()
+            String.t() | [String.t() | {:asc | :desc, String.t()}] | Keyword.t()
```

Comment:
```suggestion
            String.t() | [String.t() | {:asc | :desc, String.t()}]
```

You don't need the keyword() spec due to  it being equivalent to `[{atom(), term()}]`. Also, only strings are valid as values in this context

Reply:
Also, I think that Keyword.t() doesn't exist

Reply:
Thanks! `Keyword.t()` does exist: https://hexdocs.pm/elixir/1.12/Keyword.html#t:t/0. But you're right it's not necessary anymore!

Reply:
Only string should be valid, so that's good.

Reply:
Nice! I though only keyword() was valid :)